### ISSUE TITLE
Extend scales domain by fixed amount

### ISSUE THREAD

**Issue by hadley** [vega/vega#104]
It'd also be nice if there was some way to expand the domain of continuous scales slightly, ensuring that the data wouldn't overlap with the axes. Perhaps this could be implemented with `offset` and `mult` parameters applied to the domain.

**Comment by espinielli** [comment]
Yes it is annoying to have data point on/close on axis...I have a [little example](http://bl.ocks.org/espinielli/87047968c67c02dbcb8d) I would like to improve before going to production...

**Comment by hydrosquall** [comment]
Do we picture this spacing as something that a user would configure, or something that is added automatically?

**Comment by PBI-David** [comment]
Relevant (and popular) SO question: https://stackoverflow.com/questions/75456476/setting-a-dynamic-domain-min-and-max-axis-in-vega-lite-deneb/75456962#75456962
